test/k8s/manifests/high-scale-ipcache.yaml

```diff
@@ -108,3 +108,21 @@ spec:
   - toServices:
     - k8sService:
         serviceName: http
+---
+apiVersion: cilium.io/v2alpha1
+kind: CiliumWorldCIDRSet
+metadata:
+  name: test-world-cidrs
+spec:
+  encapsulate: false
+  cidrGroupRef:
+  - no-encap-cidrs
```

Comment:
Nit: We should make this plural to be consistent with the fact that it's `[]CIDRGroupRef`.

Reply:
totally. RIP my green CI :p.